aat/src/aat/resources/features/F-080/F-080.feature

```diff
@@ -0,0 +1,45 @@
+@F-080
+Feature: F-080: Get Case details by Jurisdiction
+
+  Background:
+    Given an appropriate test context as detailed in the test data source
+
+  @S-080.1
+  Scenario: must return 200
```

Comment:
Scenario title could be better and not mention 200 code. Like "must successfully get case type details for an available jurisdiction". just as an example..

Reply:
Done